cs251tk/toolkit/args.py

```diff
@@ -159,6 +161,19 @@ def process_args():
     parser = build_argparser()
     args = vars(parser.parse_args())
 
+    if args['ci']:
+        if not args['course'] or not args['students']:
+            print("ci flag must be accompanied by course and student flags", file=sys.stderr)
+            sys.exit(1)
+        args['highlight_partials'] = True
+        args['no_progress'] = True
+        args['no_update'] = True
+        args['no_check'] = True
+        _, string, _ = run(['ls', 'students/{}/'.format(str(args['students'][0]).replace("['", "").replace("']", ""))])
+        for line in string.split('\n'):
+            if "hw" in line or "lab" in line or "ws" in line:
+                args['to_record'] = args['to_record'] + [[line]]
```

Comment:
Let's just call `args['to_record'].append(line)` here

Reply:
It'll have to be `args['to_record'].append([line]), but sure.